aea/cli/utils/config.py

```diff
@@ -117,13 +117,13 @@ def validate_cli_config(config: Dict) -> None:
     validator.validate(config)
 
 
-def _init_cli_config() -> None:
+def write_cli_config(config: Dict) -> None:
```

Comment:
Don't change apis unnecessarily in a backward incompatible way
```suggestion
def _init_cli_config(config: Dict = DEFAULT_CLI_CONFIG) -> None:
```